bcipy/helpers/stimuli.py

```diff
@@ -227,6 +228,144 @@ def extract_trials(
         return np.stack(new_trials, 1)  # C x T x S
 
 
+class GazeReshaper:
+    def __call__(self,
+                 trial_targetness_label: List[str],
+                 inq_start_times: List[float],
+                 target_symbols: List[str],
+                 gaze_data: np.ndarray,
+                 sample_rate: int,
+                 trials_per_inquiry: int,
+                 offset: float = 0,
+                 channel_map: List[int] = None,
+                 poststimulus_length: float = 0.5,
+                 prestimulus_length: float = 0.0,
+                 transformation_buffer: float = 0.0,
+                 ) -> dict:
+        """Extract inquiry data and labels.
```

Comment:
This could use more documentation to better explain the processing flow and difference from other reshapers.